core-bundle/assets/controllers/operations-menu-controller.js

```diff
@@ -5,7 +5,7 @@ export default class OperationsMenuController extends Controller {
     static targets = ['menu', 'submenu', 'controller', 'title'];
 
     connect () {
-        if (!this.hasMenuTarget) {
```

Comment:
I don't think this is correct. You have no controller when all operations are visible, but you still need the instance for the context menu